<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidget</class>
 <widget class="QWidget" name="SettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#SettingWidget
{
    background-color: rgb(207, 231, 244);
}

QPushButton#closeBtn
{
    border: none;
    image:url(://resource/button/shut.png);

}
QPushButton#closeBtn:hover
{
    background-color: rgb(231, 37,54);
}
QWidget#widget QPushButton
{
    border: none;
    border-radius: 5px;
    background-color: rgb(6,163,220);
    min-width:80px;
	min-height:30px;
    color:white;
}
QWidget#widget QPushButton:hover
{
    background-color: rgb(16, 196, 245);
}
QStackedWidget{	
	background-color: qlineargradient(spread:pad,x1:0, y1:0, x2:0, y2:1, stop:0 rgba(232, 241, 249, 0), stop:0.1 rgba(232, 241, 249, 1), stop:0.2 rgba(232, 241, 249, 255), stop:0.7 rgba(232, 241, 249, 255),stop:1 rgba(252, 252, 252, 255));
	border-radius:5px;
}
QGroupBox{	
	border:none;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>程序设置</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>281</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>自定义桌面壁纸</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>选择壁纸</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>271</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/resource/desktopRole/summerGril/0.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="role1checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>模型①</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/resource/desktopRole/littleBoy/0.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="role2checkBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>模型②</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>人物心情</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="radioButton_1">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>44</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>开心(喜)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>71</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>哀怨(伤)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>98</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>调皮(屁)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>125</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>害羞(羞)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="role3checkBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>73</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>模型③</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>241</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/resource/desktopRole/blackGril/action1-happy/0.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>groupBox</zorder>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>98</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="setBtn">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="role1Btn">
      <property name="text">
       <string>人物模型①</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="role2Btn">
      <property name="text">
       <string>人物模型②</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="role3Btn">
      <property name="text">
       <string>人物模型③</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
